server/parsec/templates/account_email_validation.html

```diff
@@ -0,0 +1,558 @@
+                        </td>
+                      </tr>
+
+                    </table>
+                  </td>
+                </tr>
+              </table>
+            </td>
+          </tr>
+
+          <!-- Footer Section -->
+          <tr class="footer-section">
+            <td>
+                <table cellspacing="0" cellpadding="8">
+                    <tr class="top-section">
+                        <td class="closing-message-section">
+                            <table cellspacing="0" cellpadding="0" align="left">
+                                <tr>
+                                    <td>
+                                        <p>Sincerely<br />
+                                        Parsec Cloud team</p>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td>
+                                        <img style="margin-top: 12px; height: 16.8px;" src="{{ server_url }}/static/logo-parsec-row.png" alt="Parsec Cloud" title="Parsec" width="100" />
+                                    </td>
+                                </tr>
+                            </table>
+                        </td>
+                        <td class="company-links-section">
+                            <table cellspacing="0" cellpadding="0" align="right">
+                                <tr class="social-networks">
+                                    <td>
+                                        <a class="linkedin"
+                                            href="https://www.linkedin.com/company/parsec-by-scille/mycompany/">
+                                            <img src="cid:linkedin" alt="LinkedIn">
+                                        </a>
+                                        <a class="twitter" href="https://twitter.com/parsecscille">
+                                            <img src="cid:twitter" alt="Twitter">
+                                        </a>
+                                        <a class="parsec" href="https://parsec.cloud">
+                                            <img src="cid:parsec-icon" alt="Parsec">
+                                        </a>
+                                    </td>
+                                </tr>
+                                <tr class="call-to-action">
+                                    <td>
+                                        <a href="https://parsec.cloud/en/start-parsec" rel="noopener noreferrer"
+                                            target="_blank">Try Parsec for free</a>
+                                    </td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                    <tr class="bottom-section">
+                        <td colspan="2">
+                            <small>2024 • Parsec Cloud</small>
```

Comment:
Is the year really necessary here?
```suggestion
                            <small>Parsec Cloud</small>
```